### PR TITLE
Migrate d2l-action-button-group to d2l-overflow-group

### PR DESCRIPTION
**Pull request by GeorgePlukov**
I recently rewrote the `d2l-action-button-group` component to Lit and renamed it to `d2l-overflow-group`. 

The functionality matches pretty closes except for two properties:
 - `opener-type="icon"` `replaces opener-type="more"`
 - `opener-style="subtle"` has been added to provide subtle button styling (default in the d2l-action-button-group)

One minor visual change (as reflected in the visual diffs) is the icon color for the `...` being changed from ferrite to celestine.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -53,7 +53,6 @@
     "@brightspace-ui/intl": "^3.2.0",
     "@webcomponents/webcomponentsjs": "^2",
     "array-flat-polyfill": "^1.0.1",
-    "d2l-button-group": "BrightspaceUI/button-group#semver:^3",
```
Comment (Vitalii-Misechko) [inline package.json:56]: It seems that the PR requires an update of `@brightspace-ui/core` as well.

**Comment by Vitalii-Misechko** [comment]
Migration to `d2l-action-button-group` looks short and sweet. However, `d2l-overflow-group` causes multiple unit tests to fail both locally and on Sauce Lab. 
```
Error: ResizeObserver loop limit exceeded 
```

@johngwilkinson, any thoughts on this migration?

**Comment by GeorgePlukov** [comment]
Thanks for the comments. It seems like the tests are only failing occasionally as sometimes running them locally they pass fine. I am using the resize observer in the new version of this component. I'll see if I can replicate the failing tests somehow and patch it

**Comment by GeorgePlukov** [comment]
I managed to get rid of that resize observer error by wrapping the callback function in requestAnimationFrame 👍

**Review by Vitalii-Misechko — APPROVED** [review]
LGTM!

**Review by johngwilkinson — APPROVED** [review]

**Pull request merged**